__tests__/Lumberjack.spec.js

```diff
@@ -13,24 +13,25 @@
  */
 
 import util from 'util';
+import { Writable } from 'stream';
 
 import Lumberjack from '../src/Lumberjack';
 
 jest.mock('util', () => ({ format: jest.fn() }));
 
 describe('Lumberjack', () => {
-  function createStream({ write, once, removeListener } = {}) {
-    return {
+  function createStream({ write } = {}) {
+    const stream = new Writable({
       // required by Console
       // https://github.com/nodejs/node/blob/v8.11.3/lib/console.js#L34
```

Comment:
as this is no longer targeting node 8.x this comment can be updated to source code that is targeted, like
https://github.com/nodejs/node/blob/v12.0.0/lib/internal/console/constructor.js#L83

Reply:
Good catch!

Reply:
updated in 778db6e